### PR TITLE
Changelog for 4.0.0-alpha.20

### PR DESCRIPTION
**Pull request by dtassone**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,55 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.20](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.19...v4.0.0-alpha.20)
+
+###### _Feb 16, 2021_
+ 
+Big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add new selectionModel prop (#986) @dtassone
```
Comment (oliviertassinari) [inline CHANGELOG.md:12]:
```suggestion
- 🎁 Add new `selectionModel` prop (#986) @dtassone
  The prop can be used to control the selected rows in the data grid. [See the docs](https://material-ui.com/components/data-grid/selection/#controlled-selection).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,55 @@
...
+- [DataGrid] Fix scrollbar size on windows (#1061) @dtassone
+- [DataGrid] Fix warning with v5 (#1038) @oliviertassinari
+- [DataGrid] Remove showToolbar prop (#948) @DanailH
+- [DataGrid] Remove sortDirection from column definitions (#1015) @dtassone
+- [DataGrid] Resolve the api ref at the same time as any other ref (#990) @oliviertassinari
+- [DataGrid] Use the disableDensitySelector to disable the DensitySelector (#1031) @DanailH
+- [DataGrid] fix passing [] or undefined in sortModel prop (#1035) @dtassone
+- [XGrid] Fix server-side multi filters (#1029) @dtassone
+
+### docs
```
Comment (oliviertassinari) [inline CHANGELOG.md:38]:
```suggestion
### Docs
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,55 @@
...
+
+- [docs] Add code snippet for localization docs in the data grid (#1024) @DanailH
+- [docs] Fix usage of the wrong type (#1062) @oliviertassinari
+- [docs] Reduce fears around license upfront @oliviertassinari
+- [docs] Update streaming docs (#1013) @dtassone
+
+### core
```
Comment (oliviertassinari) [inline CHANGELOG.md:45]:
```suggestion
### Core
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,55 @@
...
+
+- [core] Batch small changes (#991) @oliviertassinari
+- [core] Save/restore actual yarn cache folder (#1039) @oliviertassinari
+
+### test
+
```
Comment (oliviertassinari) [inline CHANGELOG.md:51]: I think that tests are like core changes, about the infrastructure work to support the features, developers reading the changelog rarely care

```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,55 @@
...
+- 🎁 Add new selectionModel prop (#986) @dtassone
+- 🚀 Fix controllable filters and select all rows with filters (#1020) @dtassone
+- 💅 Add support for default props from theme (#1019) @DanailH
+- 🙌 Fix scrollbar size on windows (#1061) @dtassone
+- ⚡️ Fix server-side multi filters (#1029) @dtassone
+- 🐛 Polish existing features, fix 9 issues.
+
+### @material-ui/x-grid@v4.0.0-alpha.19 / @material-ui/data-grid@v4.0.0-alpha.19
+
+- [DataGrid] Add bgBG locale (#983) @DanailH
```
Comment (oliviertassinari) [inline CHANGELOG.md:21]: It's really important that we document the breaking changes: #1021, #1015, #986, #948.

```suggestion
#### Breaking changes

#### Changes

- [DataGrid] Add bgBG locale (#983) @DanailH
```

**Review by oliviertassinari — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,55 @@
...
+- [DataGrid] Fix controllable filters and select all rows with filters (#1020) @dtassone
+- [DataGrid] Fix onPageChange and onPageSizeChange event trigger (#1034) @dtassone
+- [DataGrid] Fix process is not defined (EXPERIMENTAL_ENABLED) (#1027) @leontastic
+- [DataGrid] Fix scrollbar size on windows (#1061) @dtassone
+- [DataGrid] Fix warning with v5 (#1038) @oliviertassinari
+- [DataGrid] Remove showToolbar prop (#948) @DanailH
+- [DataGrid] Remove sortDirection from column definitions (#1015) @dtassone
+- [DataGrid] Resolve the api ref at the same time as any other ref (#990) @oliviertassinari
+- [DataGrid] Use the disableDensitySelector to disable the DensitySelector (#1031) @DanailH
+- [DataGrid] fix passing [] or undefined in sortModel prop (#1035) @dtassone
```
Comment (oliviertassinari) [inline CHANGELOG.md:35]:
```suggestion
- [DataGrid] Fix passing [] or undefined in sortModel prop (#1035) @dtassone
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,55 @@
...
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.20](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.19...v4.0.0-alpha.20)
+
+###### _Feb 16, 2021_
+ 
+Big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add new selectionModel prop (#986) @dtassone
+- 🚀 Fix controllable filters and select all rows with filters (#1020) @dtassone
```
Comment (oliviertassinari) [inline CHANGELOG.md:13]:
```suggestion
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,55 @@
...
+- 💅 Add support for default props from theme (#1019) @DanailH
+- 🙌 Fix scrollbar size on windows (#1061) @dtassone
+- ⚡️ Fix server-side multi filters (#1029) @dtassone
```
Comment (oliviertassinari) [inline CHANGELOG.md:16]:
```suggestion
```

**Review by oliviertassinari — COMMENTED** [review]
How about we promote https://master--material-ui-x.netlify.app/components/data-grid/localization/#locale-text first, asking for help to the community to contribute them?

In the main l10n page, we have:

> Please do consider contributing new translations back to Material-UI by opening a pull request. However, Material-UI aims to support the 100 most popular locales, we might not accept contributions for locales that are not frequently used, for instance gl-ES that has "only" 2.5 million native speakers.

**Comment by DanailH** [comment]
> How about we promote https://master--material-ui-x.netlify.app/components/data-grid/localization/#locale-text first, asking for help to the community to contribute them?
> 
> In the main l10n page, we have:
> 
> > Please do consider contributing new translations back to Material-UI by opening a pull request. However, Material-UI aims to support the 100 most popular locales, we might not accept contributions for locales that are not frequently used, for instance gl-ES that has "only" 2.5 million native speakers.

I guess @oliviertassinari meant putting it as a highlighted item that points to the docs?

File: CHANGELOG.md (hunk 5 above)
Comment (oliviertassinari) [reply CHANGELOG.md:24]: Do you want to add the content for these changes?

**Comment by oliviertassinari** [comment]
@DanailH Yes, exactly, using it as our main highlight for the version (it was one of the most upvoted issues) and as an opportunity to ask for contributions.

**Review by oliviertassinari — APPROVED** [review]

File: CHANGELOG.md [hunk 9]
```diff
@@ -3,6 +3,95 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.20](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.19...v4.0.0-alpha.20)
+
+###### _Feb 16, 2021_
```
Comment (oliviertassinari) [inline CHANGELOG.md:8]: Something that Sebastian has raised, the heading structure isn't continuous.

```suggestion
_Feb 17, 2021_
```

**Pull request merged**